fuel/datasets/hdf5.py

```diff
@@ -106,16 +106,20 @@ class H5PYDataset(Dataset):
          array
       4. ``stop`` : stop index (exclusive) of the split in the source
          array
-      5. ``available`` : boolean, ``False`` is this split is not available
+      6. ``indices`` : h5py.Reference, reference to a dataset containing
+         subset indices for this split/source pair.
+      6. ``available`` : boolean, ``False`` is this split is not available
          for this source
-      6. ``comment`` : comment string
+      7. ``comment`` : comment string
 
     Parameters
     ----------
     file_or_path : :class:`h5py.File` or str
         HDF5 file handle, or path to the HDF5 file.
-    which_set : str
-        Which split to use.
+    which_set : str or tuple of str
```

Comment:
It is a reoccurring question in Blocks whether when a tuple or a list are possible arguments a single object also should be accepted. I guess that we are sort of converging to minimalism in the interface, which means that only an iterable should probably be accepter in this case. What's your opinion?